README.md

```diff
@@ -84,7 +84,7 @@ The *distribution of interest* (DoI) essentially specifies for which points surr
 The distribution can be specified via an implementation of the `DoI` class in the `trulens.nn.distributions` module, which is a function taking an input record and producing a list of sample input points to aggregate attribution over.
 A few common default distributions implementing the `DoI` class can be found in the `trulens.nn.distributions` module. 
 
-See [Attributions for Different Use Cases](trulens.org/attribution_parameterization/) for further explanations of the purpose of these parameters and examples of their usage.
```

Comment:
If we prepend `https://www.` to the original URL, it should work and we can point to our custom domain which is consistent with the rest of the readme/docs